org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/console/JavaStackTraceHyperlink.java

```diff
@@ -721,7 +721,18 @@ protected String getLinkText() throws CoreException {
 			if (linkStart == -1) {
 				linkStart = line.lastIndexOf('\t', regionOffsetInLine);
 			}
-
+			String extractedTrace = line.substring(linkStart == -1 ? 0 : linkStart + 1, linkEnd + 1).trim();
+			if (extractedTrace.charAt(0) == '(') {
+				int lastOpen = line.lastIndexOf('(');
+				if (lastOpen > 0) {
+					if (Character.isWhitespace(line.charAt(lastOpen - 1))) {
+						extractedTrace = line.substring(0, lastOpen - 1).trim() + line.substring(lastOpen);
+						linkStart = extractedTrace.lastIndexOf(' ', regionOffsetInLine);
+						linkEnd = extractedTrace.indexOf(')', regionOffsetInLine);
```

Comment:
shouldn't it be
```suggestion
						linkEnd = extractedTrace.indexOf(')', linkStart);
```

to prevent linkEnd < linkStart?

Reply:
Wont be needed these now. as trim() will remove all the whitespaces